cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/SynchronousMachineConversion.java

```diff
@@ -57,7 +57,11 @@ public void convert() {
         convertedTerminals(g.getTerminal());
         convertReactiveLimits(g);
         if (p.asInt("referencePriority", 0) > 0) {
-            SlackTerminal.attach(g.getTerminal().getBusBreakerView().getBus());
+            if (g.getTerminal().getBusBreakerView().getBus() == null) {
+                context.ignored("referencePriority", () -> String.format("%s is disconnected from the network", g.getId()));
```

Comment:
Even if the generator is disconnected from the network at this point, it may become eventually connected (if a switch is closed). 
Also, we could keep the information to be able to reproduce in the CGMES SSH export the same Steady State Hypothesis we have seen in the original files.